indexer/services/query/erc20.go

```diff
@@ -0,0 +1,37 @@
+package query
+
+import (
+	"github.com/ethereum-optimism/optimism/indexer/db"
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func ERC20(address common.Address, client *ethclient.Client) (*db.Token, error) {
+	contract, err := bindings.NewERC20(address, client)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
This is fine but makes me think we should vendor in multicall3 so we can pack multiple eth calls into a single request